.huskyrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  hooks: {
```

Comment:
Hmm. This should have failed the linter since it doesn't have quotes around it. Any ideas why it didn't?

```suggestion
  "hooks": {
```